common/lib/dependabot/pull_request_creator/github.rb

```diff
@@ -42,8 +42,8 @@ def initialize(source:, branch_name:, base_commit:, credentials:,
       end
 
       def create
-        return if branch_exists?(branch_name) && unmerged_pull_request_exists?
-        return if require_up_to_date_base? && !base_commit_is_up_to_date?
+        raise "Unmerged PR exists" if branch_exists?(branch_name) && unmerged_pull_request_exists?
+        raise "Base commit is not up to date" if require_up_to_date_base? && !base_commit_is_up_to_date?
```

Comment:
[newbie q] will these errors get surfaced to the user or will they be currently visible in the logs?

Reply:
These will be in the logs, but there is another effort to show the errors to the users as well.